src/cryptography/hazmat/backends/__init__.py

```diff
@@ -17,7 +17,12 @@ def _available_backends():
 
     if _available_backends_list is None:
         _available_backends_list = [
-            ep.resolve()
+            # setuptools 11.3 deprecated support for the require parameter to
```

Comment:
Can you put some string we'll remember to grep for here, so we don't forget to re-review in 6 months, maybe just use `DeprecatedIn`?